components/calendar/src/hebrew.rs

```diff
@@ -531,86 +557,35 @@ mod tests {
             (1, TEVET, 5783),
         ];
 
-        let civil_hebrew_dates: [(u8, u8, i32); 48] = [
```

Comment:
Issue: why are you removing the `civil_hebrew_dates` assertions?

Reply:
the code being tested doesn't exist anymore. It was needless complexity and the book/civil thing isn't a huge issue

Also the civil data had bugs.

Reply:
I don't understand. This table of civil Hebrew dates corresponds to the ISO dates above. We can and should test that the keviyah code produces the correct results.

Reply:
If we keep either table though I'm saying we should keep the civil table because that's the public behavior. Book Hebrew is only for internal things and isn't exposed through API. It's easier to read because we can use constants for the months since the leap month is at the end. I wrote the ordinal constants with Pedro.

Reply:
> It's easier to read because we can use constants for the months since the leap month is at the end

Right, and I think that's a valuable readability improvement to retain.

Reply:
Besides, the public behavior is with month codes.

How about I rewrite this code to use month codes instead.

Reply:
Done.

Reply:
Ok, but ordinals aren't public? They are in Temporal

Reply:
Not currently, no. Could be rather easily added in the long run.

Reply:
Well. They're public with the [individual constructors](https://unicode-org.github.io/icu4x/rustdoc/icu/calendar/struct.Date.html#method.try_new_hebrew_date_with_calendar).

I could use them but I'd rather use month codes because then I can use constants without needing funky ordinal-fixing math.

Reply:
And also via FormattableMonth

I still think it's valuable to test that the ordinals are what they are supposed to be over a range of several years that includes both leap and regular months. I still don't understand why you want to delete them. Is the readability of the cases the only issue? Or is there an actual bug?

Reply:
I was hoping to keep them in earlier and added some asserts about them that seemed to fail. Those asserts tested code I was deleting anyway so I deleted them. I didn't spend effort verifying but I did verify the const based data was correct.

I do not think that table is useful from a maintenance or review standpoint. I am willing to add a test for ordinal construction that manually does the leap year month offsetting for the known leap years in the data (I think there's only one) and then uses the same data. I don't want us to have two tables, that's more inscrutable bags of test data that we don't need.

Reply:
Would you be open to merging the three arrays into a single array of tuples?

Reply:
I'm planning on merging the two arrays. I'd rather not re add the manual ordinal one. I'll do the thing I proposed where I test ordinal code via checking for known leap years.

Reply:
I was happy to have the ordinal constants back when we had the whole civil/book thing and plenty of ways that conversion could go wrong. Now that that's completely gone, I would rather have this one table.

Reply:
I've added tests for ordinal stuff and cleaned up the tables.